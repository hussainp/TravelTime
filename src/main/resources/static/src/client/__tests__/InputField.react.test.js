'use strict';

import React from 'react';
import TextInput from '../app/helpercomponents/textInput.jsx';
import renderer from 'react-test-renderer';

it('textInput with text type', () => {
    const component = renderer.create(
        <TextInput type='text' value='SAN' label='Location' name='Location'
htmlFor='Location' isRequired='true' onChange='onChangeLocation' messageRequired={'Location Required'}
list='locations'
    />
);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
});

it('textInput with date type', () => {
    const component = renderer.create(
    <TextInput type="date" value='2016-09-23' label='Check-In' name='CheckIn'
htmlFor='CheckIn' isRequired='true' onChange='onChangeFromDate'
messageRequired={'Check-in required'}
min='2016-09-20'
    />
);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
});