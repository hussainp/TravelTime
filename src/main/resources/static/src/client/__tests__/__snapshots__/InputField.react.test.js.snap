exports[`test textInput with date type 1`] = `
<div
  className="col-sm-6 ">
  <label
    htmlFor="CheckIn">
    CheckIn
  </label>
  <input
    className="form-control"
    data-type="date"
    id="CheckIn"
    min="2016-09-20"
    name="CheckIn"
    onChange={[Function bound handleChange]}
    placeholder="CheckIn"
    required="true"
    type="date"
    value="2016-09-23" />
  <span
    className="error  errormsg" />
</div>
`;

exports[`test textInput with text type 1`] = `
<div
  className="col-sm-6 ">
  <label
    htmlFor="Location">
    Location
  </label>
  <input
    className="form-control"
    id="Location"
    list="locations"
    name="Location"
    onChange={[Function bound handleChange]}
    placeholder="Location"
    required="true"
    value="SAN" />
  <span
    className="error  errormsg" />
</div>
`;
